test2code-admin/src/main/kotlin/com/epam/drill/plugins/test2code/Methods.kt

```diff
@@ -36,7 +36,7 @@ data class Method(
     val name: String,
     val desc: String,
     val hash: String,
-    val lambdasHash: Map<String, String> = emptyMap(),
+    val annotations: List<String>
```

Comment:
Why not add `= emptyList()` so as not to create boilerplate code that Method object was initialized

Reply:
added.